### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by codephile1221**
Just a few changes. Feel free to discard if felt unnecessary.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,9 +26,9 @@ and then start working on it.
 
 If you find a bug in the code or a problem while running the code, please open
 an issue and provide some details like how to reproduce it. If you feel some
-feature is miising or the documentation lacks in some way, feel free to open an
```
Comment (codephile1221) [inline CONTRIBUTING.md:30]: fixed minor typo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,9 +26,9 @@ and then start working on it.
...
+feature is missing or the documentation lacks in some way, feel free to open an
 issue in this case too. If you are able to solve the issue, sending a pull
-request is encouraged rather than just opening an issue.
```
Comment (codephile1221) [inline CONTRIBUTING.md:32]: Improved tone so as to not come across as reprimanding.

**Comment by codephile1221** [comment]
Hello J-tesla,
Made a few changes to CONTRIBUTING.md. Feel free to discard them if you feel they are unnecessary.

**Comment by j-tesla** [comment]
Hey @codephile1221 that's cool. Can you please push the extra commits from restyled/pull-20 to this PR; or fix the style issue yourself. So that I can merge this PR instead of #21.

**Comment by codephile1221** [comment]
Sure.

**Comment by codephile1221** [comment]
Done!
Thanks for giving me the opportunity to contribute

**Pull request merged**